src/main-oe/sgxlkl_run_oe.c

```diff
@@ -816,235 +898,112 @@ void set_tls(sgxlkl_config_t* conf)
         // WRFSBASE instruction can always be used inside the enclave. It will
         // either be a legal instruction, or it will be emulated by handling
         // the SIGILL in `oecore` first-pass exception handler.
-        conf->fsgsbase = 1;
+        econf->fsgsbase = 1;
```

Comment:
Not sure what you mean. The renaming of the variable doesn't affect attestation, but `fsgsbase` is an attested setting. Do you have reason to believe attestation could fail (e,g. OE will segfault) when we run with `econf->fsgsbase == 0`?

Reply:
It could be that I don't understand the code, but it looks like the launcher patches the enclave config here, and as far as I know we're only supposed to do that in debug/non-release mode, otherwise it's hard to predict the eeid mrenclave.

Reply:
Oh yeah, good catch! There's no env var for this one so I missed it. I'll have to disable this kind of auto-detection when we're using JSON configs.